runtime/doc/lua.txt

```diff
@@ -1805,15 +1805,16 @@ vim.system({cmd}, {opts}, {on_exit})                            *vim.system()*
         (`vim.SystemObj`) Object with the fields:
         • cmd (string[]) Command name and args
         • pid (integer) Process ID
-        • wait (fun(timeout: integer|nil): SystemCompleted) Wait for the
-          process to complete. Upon timeout the process is sent the KILL
-          signal (9) and the exit code is set to 124. Cannot be called in
-          |api-fast|.
-          • SystemCompleted is an object with the fields:
-            • code: (integer)
-            • signal: (integer)
-            • stdout: (string), nil if stdout argument is passed
-            • stderr: (string), nil if stderr argument is passed
+        • wait (fun(timeout: integer|nil): vim.SystemCompleted) Wait for the
+          process to complete, for {timeout} milliseconds (if {timeout} is
+          given) or the timeout specified in {opts}. Upon timeout the process
+          is sent the TERM signal (15) and the exit code is set to 124. Cannot
```

Comment:
Off topic, why does it set code=124 on timeout, instead of `-1` (like `jobwait()`)? I assume that `-1` has less chance of conflicting with an exitcode emitted by the child.

Reply:
124 is the same exit code used by the `timeout` bash command which I think is a much better choice to `-1`.